biofefi/machine_learning/call_methods.py

```diff
@@ -14,6 +14,65 @@
 from biofefi.options.plotting import PlottingOptions
 
 
+def plot_confusion_matrix(
+    estimator,
+    X,
+    y,
+    set_name: str,
+    model_name: str,
+    directory: Path,
+    plot_opts: PlottingOptions | None = None,
+):
+    """
+    Plot the confusion matrix for a multi-class or binary classification model.
+
+    Args:
+        estimator: The trained model.
+        X: The features.
+        y: The true labels.
+        set_name: The name of the set (train or test).
+        model_name: The name of the model.
+        directory: The directory path to save the plot.
+        plot_opts: Options for styling the plot. Defaults to None.
+
+    Returns:
+        None
+    """
+
+    plt.style.use(plot_opts.plot_colour_scheme)
+
+    disp = ConfusionMatrixDisplay.from_estimator(
+        estimator=estimator,
+        X=X,
+        y=y,
+        normalize=None,
+        colorbar=False,
+        cmap="Grays",
+    )
+
+    disp.ax_.set_xlabel(
+        "Predicted label",
+        fontsize=plot_opts.plot_axis_font_size,
+        fontfamily=plot_opts.plot_font_family,
+    )
+    disp.ax_.set_ylabel(
+        "True label",
+        fontsize=plot_opts.plot_axis_font_size,
+        fontfamily=plot_opts.plot_font_family,
+    )
+
```

Comment:
Can you add some code to set the `x` and `y` ticks' angles please? Check out `plot_lime_importance` for an example of how to do that.